### PR TITLE
Add a switch for the firewall tasks

### PR DESCRIPTION
**Pull request by triat**
The principal goal of this pull request is to give the choice to the user to activate or not the installation and configuration of UFW.

The reasons that pushed me to propose this change are:
- Some systems come with `nftables` and this is less a pain to manage than `iptables`, less reason to have a frontend wrapper (`ufw`) on top of it
- Some systems come with their own preconfigured rules. Those still need to be defined on ansible to keep track of them but this might not be done on `ufw`.

I kept the tasks `firewall.yml` on by default, to keep the same behavior as this is now but added a switch to include the firewall tasks on the `main.yml` so there is a possibility to manage it differently.

Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,8 +205,8 @@ ufw_outgoing_traffic:
   - 443
   - 853
 ```
-
-Open `ufw` ports, allowing outgoing traffic.
+`ufw_enable: false` gives you the opportunity to manage your own firewall and firewall rules.
```
Comment (konstruktoid) [inline README.md:208]: I think this needs a rewording, to reduce any confusion we should probably describe what the default does.
"`ufw_enable: true` enables `ufw` and related firewall rules. If you want to manage your own firewall and firewall rules, this should be `false`" or something similar.

File: tasks/main.yml [hunk 2]
```diff
@@ -14,6 +14,7 @@
     apply:
```
Comment (konstruktoid) [inline tasks/main.yml:14]: i can fix this later, but `firewall.yml` should be renamed `ufw.yml` and `name: install and configure ufw`

**Review by konstruktoid — CHANGES_REQUESTED** [review]

**Comment by konstruktoid** [comment]
Makes sense, I've added a couple of comments.

File: tasks/main.yml (hunk 2 above)
Comment (triat) [reply tasks/main.yml:14]: This was an easy change, I've done it quickly. I should have renamed every place where this was using the word "firewall" when this means "ufw".

File: README.md (hunk 1 above)
Comment (triat) [reply README.md]: I've tried something, I'll let you review it and let me know if this is more clear now. English not being my mother tongue, my sentence can undoubtedly be improved, especially when this comes to documentation :)

File: tasks/main.yml (hunk 2 above)
Comment (konstruktoid) [reply tasks/main.yml:14]: Awesome, thanks.
I'll run some tests later today.

**Review by konstruktoid — APPROVED** [review]

**Comment by konstruktoid** [comment]
Sorry for the delay.

**Pull request merged**